partner_external_maps/partner_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2015 Akretion France (www.akretion.com)
+    @author: Alexis de Lattre <alexis.delattre@akretion.com>
+    The licence is in the file __openerp__.py
+-->
+
+<openerp>
+<data>
+
+<!--  Inherit partner view to add 'Map' button -->
+<record id="view_partner_form" model="ir.ui.view">
+    <field name="name">map.button.res.partner.form</field>
+    <field name="model">res.partner</field>
+    <field name="inherit_id" ref="base.view_partner_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//div[@name='buttons']" position="inside">
+            <button class="oe_inline oe_stat_button"
```

Comment:
Isn't better to provide these buttons next to the street information or at least inside the same tab instead of being mixed with the bunch of buttons that is already present on the partner in the smart-buttons area?